stable/rethinkdb/templates/rethinkdb-secrets.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    app: {{ template "name" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+type: Opaque
+data:
+  rethinkdb-password: {{ .Values.rethinkdbPassword | b64enc | quote }}
```

Comment:
that looks like a bit of a blocker, and so far with no solution. 
@prydonius is this a blocker this chart to be released ? how are other databases like cassandra/mongodb dealing with the issue ?